Source/XUnitExtensions/FileTestCase.cs

```diff
@@ -1,11 +1,11 @@
 using System;
 using System.Collections.Generic;
-using System.ComponentModel;
 using System.Threading;
 using System.Threading.Tasks;
 using Xunit;
 using Xunit.Abstractions;
 using Xunit.Sdk;
+using SkipException = Xunit.SkipException;
```

Comment:
Does this mean we can drop `Xunit.SkippableFact` from the csproj now (because it's part of xunit core now)?

Reply:
If I remove that package reference the. I have to replace `SkippableTestMessageBus` and `SkippableFactTestCase` with something and I'm not entirely sure what. If you're already aware of the new API, could you make that change in a follow-up PR?